tests/test_conditionals_artem.py

```diff
@@ -0,0 +1,30 @@
+import pytest
+
+from codecademy.conditionals_and_control_flow import greater_less_equal_5
+
+#
+# def test_greater_less_equal_5_minus_1():
+#     assert greater_less_equal_5(-1) == -1
+#
+#
+# def test_greater_less_equal_5_6():
+#     assert greater_less_equal_5(6) == 1
+#
+#
+# def test_greater_less_equal_5_5():
+#     assert greater_less_equal_5(5) == 0
+
+# -&     -10000000 ..... -1  0  +1 ...  2  ..4 5 6...... 100000 ..... +&
```

Comment:
please remove all the commented out code

Reply:
Why?